packages/callout-block/src/components/CalloutIcon.tsx

```diff
@@ -0,0 +1,34 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { IconInfo20, IconLightbulb20, IconMegaphone20 } from '@frontify/fondue';
+import { joinClassNames } from '@frontify/guideline-blocks-shared';
+import { ReactElement } from 'react';
+import { Icon } from '../types';
+
+type CalloutIconProps = {
+    iconType: Icon;
+    hasValue: boolean;
+    iconUrl?: string;
+};
+export const CalloutIcon = ({ iconType, hasValue, iconUrl }: CalloutIconProps) => (
```

Comment:
Nit: To me the naming of `hasValue` seems a bit confusing. `isActive` or something similiar would probably make more sense.